analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Tests/VerifierTest.cs

```
@@ -124,19 +126,19 @@ public class VerifierTest
 #if NET
 
         [TestMethod]
-        public void Constructor_RazorWithAssociatedCS() =>
+        public void Verify_RazorWithAssociatedCS() =>
             DummyCS.AddPaths(WriteFile("File.razor", """<p @bind="pValue">Dynamic content</p>"""))
                 .AddPaths(WriteFile("File.razor.cs", """public partial class File { string pValue = "The value bound"; int a = 42;  }"""))
-                .Invoking(x => x.Build()).Should().NotThrow();
+                .Invoking(x => x.Verify()).Should().Throw<UnexpectedDiagnosticException>();
```

Why it should throw the `UnexpectedDiagnosticException`?